services/federator/src/Federator/Util.hs

```diff
@@ -21,19 +21,8 @@ module Federator.Util
 where
 
 import Data.Aeson as Aeson
+import qualified Data.Char as Char
 import Imports
 
-dropPrefix :: String -> String -> Maybe String
-dropPrefix pfx str =
-  if length pfx > length str
-    then Nothing
-    else case splitAt (length pfx) str of
-      (pfx', sfx) ->
-        if pfx' /= pfx
-          then Nothing
-          else Just sfx
-
--- | This is a partial function; totality of all calls must be verified by roundtrip tests on
--- the aeson instances involved.
-wireJsonOptions :: String -> Options
-wireJsonOptions pfx = defaultOptions {fieldLabelModifier = fromJust . dropPrefix pfx . fmap toLower}
+wireJsonOptions :: Options
+wireJsonOptions = defaultOptions {fieldLabelModifier = camelTo2 '_' . dropWhile (not . Char.isUpper)}
```

Comment:
This is the same as the one in wire-api-federation. Re-use that definition?

Reply:
Yeah, I will do that.